more/app-more.js

```diff
@@ -14,6 +14,7 @@
 //    
 (function() {
     var ROOT = Script.resolvePath('').split("app-more.js")[0];
+    var DEV_PARAMETER = Script.resolvePath('').split("?")[1];
     var APP_NAME = "MORE...";
     var APP_URL = ROOT + "more.html";

```

Comment:
```suggestion
    var APP_URL = ROOT + "more.html" + (DEV_PARAMETER === "dev" ? "?dev" : "");
```
Alternative to the `if (DEV_PARAMETER) {…}` block, below.

Or better, yet, to handle Windows file:/// paths that use "\"s (which are converted to %5Cs that `tablet.gotoWebScreen()` can't handle...
```
var APP_URL = (ROOT + "more.html" + (DEV_PARAMETER === "dev" ? "?dev" : "")).replace(/%5C/g, "/");
```